test/integration/api/submissions.js

```diff
@@ -4061,9 +4062,9 @@ one,h,/data/h,2000-01-01T00:06,2000-01-01T00:07,-5,-6,,ee,ff
               body[1].instanceName.should.equal('custom name');
 
               body[0].deviceId.should.equal('updateDevice');
-              body[0].userAgent.should.equal('central/tests');
               (body[1].deviceId == null).should.equal(true);
-              body[1].userAgent.should.equal('node-superagent/3.8.3');
+              body[0].userAgent.should.equal('central-tests/2');
+              body[1].userAgent.should.equal('central-tests/1');
```

Comment:
Does this order look correct?

Reply:
I guess the more recent submission version is returned first? It seems consistent with how it was before.